doozer/doozerlib/cli/scan_fips.py

```diff
@@ -96,9 +97,18 @@ def get_all_latest_builds(self):
         for tag in brew_tags:
             builds += self.get_tagged_latest(tag=tag)
 
-        nvrs = []
+        # Find the latest build nvr per image across all tags
+        latest_build_map = {}
         for build in builds:
-            nvrs.append(build["nvr"])
+            image_name = build["name"]
+            completion_time = datetime.strptime(build["completion_time"], "%Y-%m-%d %H:%M:%S.%f")
+
+            if image_name not in latest_build_map or completion_time > latest_build_map[image_name]["time"]:
+                latest_build_map[image_name] = {"nvr": build["nvr"], "time": completion_time}
```

Comment:
Since we are not using or printing out the `time`field, we can exclude them from being stored in the dict itself right

Reply:
It's used to compare the `completion_time` of the current build and the previously stored newest build. Or am I misunderstanding what you mean?

Reply:
yes - but after its being compared, its being stored int he dict right?

`latest_build_map[image_name] = {"nvr": build["nvr"], "time": completion_time}`

since the `time` field is not used after the comparison, do we need to store it unless we're using it in some way?

Reply:
ohh my bad - I missed the `latest_build_map[image_name]["time"]` part